.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Code of Conduct
```

Comment:
IMO adding a code of conduct is something that should really only be done by project maintainers, and isn't something that should be taken lightly. I'm fine with using the Contributor Covenant and will merge this as is, but I would have preferred to see some discussion first before submitting such a change.